code/modules/economy/ATM.dm

```diff
@@ -57,7 +57,12 @@ log transactions
 	else if(authenticated_account)
 		if(istype(I,/obj/item/spacecash))
 			//consume the money
-			authenticated_account.money += I:worth
+			if(I:counterfeit)
```

Comment:
Can you remove the `:` usage entirely? Do proper typecasting and then just use the `.` access operator.